course/layouts/partials/course-image-url.html

```diff
@@ -0,0 +1,8 @@
+{{- $courseData := .Site.Data.course -}}
+{{- $courseImageUid := $courseData.course_image.content -}}
+{{- $courseImageMetadata := (dict "Params" (dict "file" "None" "image_metadata" (dict "image-alt" "" "caption" ""))) -}}
```

Comment:
While you're making this change, can you add the image metadata to course/layouts/index.coursedata.json?

Reply:
sure, will do